client/stylesheets/_responsive.scss

```diff
@@ -387,6 +480,17 @@
     }
 }
 
+@media (max-width: 321px) {
+
+    .homeCardLi {
+        width: 110px;
+    }
+
+    .homeCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardImg` should be written in lowercase with hyphens